app/assets/javascripts/components/story/ExpandedStory/index.js

```diff
@@ -58,6 +62,13 @@ export const ExpandedStory = ({
         onEdit={(newAttributes) => editStory(story.id, newAttributes)}
       />
 
+      <ExpandedStoryTask
+        story={story}
+        onUpdate={(task, status) => updateTask(project.id, story, task, status) }
+        onDelete={(taskId) => deleteTask(project.id, story, taskId) }
+        onSave={(newAttributes) => createTask(project.id, story.id, newAttributes) }
```

Comment:
I think `newAttributes` here is only the new task, right? Can we call it `newTask` or `taskAttributes`?

Reply:
I think it could be just `task`, since here : 
```
export const createTask = (projectId, storyId, newAttributes) => {
  return (dispatch, getState, { Task }) => {
    return Task.post(projectId, storyId, newAttributes)
```

is a post for a new task and not an update.

Reply:
taskAttributes it's better!

Reply:
cool!